### ISSUE TITLE
Add structured data for New START report

### ISSUE THREAD

**Issue by awesomephant** [Peace-Research-Institute-Frankfurt/websites#99]
https://nuclearforces.org/country-profiles/russia

**Comment by fifilz** [comment]
Ich komme leider nicht weiter. Habe die Datei erstellt und "commit" durchgeführt. Wenn ich "git push" eingebe, gibt es die Rückmeldung: "remote: Repository not found. / fatal: repository 'https://github.com/awesomephant/eunpdc-demo.git/' not found".
Weißt du, woran das liegt, @awesomephant?

**Comment by awesomephant** [comment]
@fifilz yeah, `eunpdc-demo` war unser altes repo, das es nicht mehr gibt. Du kannst das aktuelle (dieses hier) mit `git clone https://github.com/Peace-Research-Institute-Frankfurt/websites.git` herunterladen - da sollte der git push funktionieren.

**Comment by fifilz** [comment]
Das glaubte ich eigentlich, gemacht zu haben. Wenn ich es noch einmal versuche, sieht das so aus:
```
PS C:\Users\filzinger\Documents\filzinger\eunpdc-demo> git clone https://github.com/Peace-Research-Institute-Frankfurt/websites.git
fatal: destination path 'websites' already exists and is not an empty directory.
PS C:\Users\filzinger\Documents\filzinger\eunpdc-demo>
```
Wieso öffnet VS Code überhaupt immer default-mäßig "eunpdc-demo" und lässt mich nicht woanders hin navigieren? 

Vielleicht wär's hilfreich, das nochmal kurz per Call zu klären, wenn du dazu Zeit hast.
